libs/dh/metering-point/feature-overview/src/lib/dh-customer-overview.component.ts

```diff
@@ -0,0 +1,117 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { Component } from '@angular/core';
+import { TranslocoDirective } from '@ngneat/transloco';
+
+import { WATT_CARD } from '@energinet-datahub/watt/card';
+import { DhEmDashFallbackPipe } from '@energinet-datahub/dh/shared/ui-util';
+import { VaterStackComponent } from '@energinet-datahub/watt/vater';
+import { WattIconComponent } from '@energinet-datahub/watt/icon';
+import {
+  WattDescriptionListComponent,
+  WattDescriptionListItemComponent,
+} from '@energinet-datahub/watt/description-list';
+
+@Component({
+  selector: 'dh-customer-overview',
+  imports: [
+    TranslocoDirective,
+
+    VaterStackComponent,
+    WATT_CARD,
+    WattIconComponent,
+    WattDescriptionListComponent,
+    WattDescriptionListItemComponent,
+    DhEmDashFallbackPipe,
+  ],
+  styles: `
+    :host {
+      display: block;
+    }
+
+    .customer {
+      width: 500px;
+    }
+
+    .protected-address {
+      background: var(--watt-color-secondary-ultralight);
+      color: var(--watt-color-neutral-grey-800);
+      border-radius: 12px;
+      display: inline-flex;
+    }
+
+    .info-text {
+      color: var(--watt-on-light-medium-emphasis);
+      margin: 0;
+    }
+
+    .show-cpr-button {
+      background-color: var(--watt-color-neutral-white);
+      border: 1px solid var(--watt-color-neutral-grey-600);
+      border-radius: 2px;
+      color: var(--watt-on-light-medium-emphasis);
+      padding: var(--watt-space-xs) var(--watt-space-s);
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
+  `,
+  template: `
+    <watt-card *transloco="let t; read: 'meteringPoint.overview.customer'" class="customer">
+      <watt-card-title>
+        <h3>{{ t('title') }}</h3>
+      </watt-card-title>
+
+      <div
```

Comment:
Comment: @PernilleChristiansen said we should just use badge

Reply:
It looks a bit ugly compared to Figma, but that's what we got :) 
One limitation is that the badge does not support icons. Projecting an icon doesn't look good. So we should only stick to text

![image](https://github.com/user-attachments/assets/a64bc70c-ea50-4216-a8a1-b7503a4c8dda)

Reply:
Let's continue with the custom component.